src/chunk_append/chunk_append.c

```diff
@@ -119,6 +120,8 @@ ts_chunk_append_path_create(PlannerInfo *root, RelOptInfo *rel, Hypertable *ht,
 		List *nested_children = NIL;
 		bool has_scan_childs = false;
 
+		Assert(ht->space->num_dimensions <= 2);
```

Comment:
What happens if the table has more than 2 space dimensions?

Reply:
then you won't get ordered append plan, this is checked properly in caller that's why it's only asserted there